Resources/views/Collector/db.html.twig

```diff
@@ -1,23 +1,45 @@
-        {% endif %}
-        {% if collector.invalidEntityCount > 0 %}
-            <span class="sf-toolbar-info-piece-additional sf-toolbar-status sf-toolbar-status-red">{{ collector.invalidEntityCount }}</span>
-        {% endif %}
-        {% if collector.cacheEnabled %}
-            <span class="sf-toolbar-info-piece-additional sf-toolbar-status sf-toolbar-status-green" title="Second level cache enabled">2l cache</span>
+        {% if profiler_markup_version == 1 %}
+
+            <img width="20" height="28" alt="Database" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAcCAYAAABh2p9gAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAQRJREFUeNpi/P//PwM1ARMDlcGogZQDlpMnT7pxc3NbA9nhQKxOpL5rQLwJiPeBsI6Ozl+YBOOOHTv+AOllQNwtLS39F2owKYZ/gRq8G4i3ggxEToggWzvc3d2Pk+1lNL4fFAs6ODi8JzdS7mMRVyDVoAMHDsANdAPiOCC+jCQvQKqBQB/BDbwBxK5AHA3E/kB8nKJkA8TMQBwLxaBIKQbi70AvTADSBiSadwFXpCikpKQU8PDwkGTaly9fHFigkaKIJid4584dkiMFFI6jkTJII0WVmpHCAixZQEXWYhDeuXMnyLsVlEQKI45qFBQZ8eRECi4DBaAlDqle/8A48ip6gAADANdQY88Uc0oGAAAAAElFTkSuQmCC" />
+                <span class="sf-toolbar-value sf-toolbar-status {% if collector.querycount > 50 %}sf-toolbar-status-yellow{% endif %}">{{ collector.querycount }}</span>
+                {% if collector.querycount > 0 %}
+                    <span class="sf-toolbar-info-piece-additional-detail">in {{ '%0.2f'|format(collector.time * 1000) }} ms</span>
+                {% endif %}
+                {% if collector.invalidEntityCount > 0 %}
+                    <span class="sf-toolbar-info-piece-additional sf-toolbar-status sf-toolbar-status-red">{{ collector.invalidEntityCount }}</span>
+                {% endif %}
+
+        {% else %}
+
+            {% if collector.querycount > 0 %}
+                {% set status = collector.invalidEntityCount > 0 ? 'red' : collector.querycount > 50 ? 'yellow' : '' %}
+
+                <svg version="1.1"xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
+                    <path fill="#AAAAAA" d="M5,8h14c1.7,0,3-1.3,3-3s-1.3-3-3-3H5C3.3,2,2,3.3,2,5S3.3,8,5,8z M18,3.6c0.8,0,1.5,0.7,1.5,1.5S18.8,6.6,18,6.6s-1.5-0.7-1.5-1.5S17.2,3.6,18,3.6z M19,9H5c-1.7,0-3,1.3-3,3s1.3,3,3,3h14c1.7,0,3-1.3,3-3S20.7,9,19,9z M18,13.6
+                    c-0.8,0-1.5-0.7-1.5-1.5s0.7-1.5,1.5-1.5s1.5,0.7,1.5,1.5S18.8,13.6,18,13.6z M19,16H5c-1.7,0-3,1.3-3,3s1.3,3,3,3h14c1.7,0,3-1.3,3-3S20.7,16,19,16z M18,20.6c-0.8,0-1.5-0.7-1.5-1.5s0.7-1.5,1.5-1.5s1.5,0.7,1.5,1.5S18.8,20.6,18,20.6z"/>
+                </svg>
+
+                <span class="sf-toolbar-value">{{ collector.querycount }}</span>
+                <span class="sf-toolbar-info-piece-additional-detail">
+                    <span class="sf-toolbar-label">in</span>
+                    <span class="sf-toolbar-value">{{ '%0.2f'|format(collector.time * 1000) }}</span>
```

Comment:
I've tweaked the display of information a bit:

1) If there are no Doctrine errors and no queries, you see nothing:

![no_queries_no_errors](https://cloud.githubusercontent.com/assets/73419/9002536/bbdaf554-3765-11e5-80e4-38092605d044.png)

2) If there are queries but no errors, you see num of queries + time:

![queries_but_no_errors](https://cloud.githubusercontent.com/assets/73419/9002545/c6b380e0-3765-11e5-8d21-3e93ec66d2df.png)

3) If there are no queries but some errors, you clearly see that:

![no_queries_but_errors](https://cloud.githubusercontent.com/assets/73419/9002552/d4253ca0-3765-11e5-8644-005c8c1a0eca.png)

4) If there are both queries and errors, you see the num of queries + time in a red background:

![queries_and_errors](https://cloud.githubusercontent.com/assets/73419/9002562/e19627b4-3765-11e5-8d79-eeb10bed91f3.png)

The expanded information contains details about the errors:

![queries_and_errors_expanded](https://cloud.githubusercontent.com/assets/73419/9002567/eab8a0f6-3765-11e5-9967-9ef9a34f4707.png)